README.md

```diff
@@ -1 +1,70 @@
-# covid-19
\ No newline at end of file
+# COVID-19
+
+In this project, I built a React Redux COVID-19 Metrics data application that shows infected cases of COVID-19 world wide.
+I got the data from a free <a href="https://covid19api.com/">Covid-19 API</a> online.
+
+### Mobile Screenshot
+
+![image](/src/images/mobile-shot.PNG)
+
+#### Desktop Screenshot
+
+![image](/src/images/pc-shot.PNG)
+
+## Built With
+
+- React
+
+- Redux
+
+- ESLint
+
+- Jest
+
+- Tailwind CSS
+
+- Framer Motion
+
+- ReactIcons
+
+## Live Demo
+
+<a href="https://effervescent-puffpuff-a0122d.netlify.app/">COVID-19 Metrics Data</a>
+
+## Live Demo Video
+
+<a href="https://www.loom.com/share/623fb8fe2a3b4dda919e4e5056fc11d0">COVID-19- Video Demo</a>
```

Comment:
- [ ] Kindly notice that the video contains a highlight about a piece of the code the student finds interesting or is proud of making.